Automattic/vip-go-mu-plugins
# Update vip-dashboard submodule to c08acac

## Description


Update vip-dashboard submodule to c08acac: dependencies security updates.

## Changelog Description


### VIP Dashboard: updated dependencies

We have updated some dependencies in vip-dashboard that address multiple security issues.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Check out PR.
2. Run `npm install` on `vip-dashboard`.
3. Build the JS changes.
4. Go to the VIP dashboard in `wp-admin`.
